app/views/games/show.html.erb

```diff
@@ -1,22 +1,18 @@
-<div id='chessboard'>
+<div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 no-padding">
 	<%= hidden_field :game_id, '', name: nil, id: nil, data: { game_id: @game.id } %>
```

Comment:
What do you mean that the 'chessboard selector wasn't defined'? I had a rule for it in games.css.scss, if nothing else. On your second point, you could re-declare Bootstrap classes in your stylesheet to add rules to them, but that doesn't sound like a clean practice to me.

I don't want to fuss much over this. I simply desire for there to be obvious labels of some sort, ids or classes, that clearly identify elements (divs especially, since they hold no inherent semantic meaning) for what they are in the markup. And it seemed straightforward to me that you could simply apply padding: 0 to selectors like 'chessboard', 'board-row', etc. Does that make sense, or am I missing something?